helm-charts/chatqna/nv-values.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Default values for chatqna.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: opea/chatqna:latest
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  # tag: "1.0"
+
+port: 8888
+service:
+  type: ClusterIP
+  port: 8888
+
+securityContext:
+  readOnlyRootFilesystem: true
+  allowPrivilegeEscalation: false
+  runAsNonRoot: true
+  runAsUser: 1000
+  capabilities:
+    drop:
+    - ALL
+  seccompProfile:
+    type: RuntimeDefault
+
+# To override values in subchart tgi
+tgi:
+  LLM_MODEL_ID: Intel/neural-chat-7b-v3-3
+  # LLM_MODEL_ID: /data/OpenCodeInterpreter-DS-6.7B
+  image:
+    repository: ghcr.io/huggingface/text-generation-inference
+    tag: "2.0"
+  resources:
+    limits:
+      nvidia.com/gpu: 1
+
+global:
+  http_proxy: ""
+  https_proxy: ""
+  no_proxy: ""
+  HUGGINGFACEHUB_API_TOKEN: ""
```

Comment:
please keep the fake 'insert-your-huggingface-token-here' as the same as gaudi-values. Otherwise, future CI test will fail

Reply:
Done